docs/common/prereq-items.md

```diff
@@ -1,17 +0,0 @@
-be sure someone in your Slack workspace has completed
```

Comment:
Removing this means the markdown_include extension is no longer needed.

Reply:
great! I'm all for removing deps